ui/app/templates/partials/replication/enable.hbs

```diff
@@ -141,10 +141,11 @@
         </select>
       </div>
       {{#if (eq mode 'secondary')}}
-        <AlertInline
-          @type="warning"
-          @message="This will immediately clear all data in this cluster!"
-        />
+          <AlertInline
```

Comment:
I think there's a few places we use this `AlertInline` in the replication partials - we should check the others too. @joshuaogle what do you think about making this red instead of yellow? I know it's a warning, but it's pretty important.

Reply:
Red should be saved for errors, especially correcting user behavior. We could maybe move it to a banner if we want more visibility though.

Reply:
I'm going to keep this the way it is now since I think it's prominent enough as-is, but we can easily change it down the line if users are repeatedly confused.

Reply:
So it looks like there's one other `AlertInline` that needs `has-top`

<img width="1408" alt="screen shot 2018-11-27 at 3 26 00 pm" src="https://user-images.githubusercontent.com/39469/49112729-26193f00-f259-11e8-9365-fc21cdca3693.png">

The force promotion one there ^.

Reply:
oh, good catch! fixing now :D